app/dts/bindings/behaviors/zmk,behavior-macro.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: Macro
+
+compatible: "zmk,behavior-macro"
+
+include: zero_param.yaml
+
+properties:
+  bindings:
+    type: phandle-array
+    required: true
+  sleep:
```

Comment:
`delay` or `delay-ms` (to match the other places where things have a -ms suffix) might be easier to understand. The code is sleeping the thread for that long, but that's an implementation detail.